lambda/simplif.ml

```diff
@@ -103,7 +103,7 @@ let simplify_exits lam =
   (* Count occurrences of (exit n ...) statements *)
   let exits = Hashtbl.create 17 in
 
-  let try_depth = ref 0 in
+  let try_depth = 0 in
```

Comment:
I think you should remove completely this line. This will give you a better idea of what still needs to be fixed, by looking at the compiler's error messages.
However, since we've only focused on `count` for now, and `try_depth` is also used in the `simplif` function below, I would suggest moving the original `let try_depth = ref 0 in` after the `count` and `count_default` functions and before the `simplif` function. This way the `simplif` function doesn't have to be changed at all.

Reply:
okay, I think it's better to go in steps only. Thanks for your feedback, will try to resolve it.